docs/minio-admin-complete-guide.md

```diff
@@ -16,6 +16,7 @@ top         provide top like statistics for MinIO
 trace       show http trace for MinIO server
 console     show console logs for MinIO server
 prometheus  manages prometheus config
```

Comment:
Looks like sub-command formatting is off..

Reply:
Thanks @harshavardhana - fixed